config/database.yml

```diff
@@ -2,8 +2,8 @@ default: &default
   adapter: mysql2
   encoding: utf8
   username: root
-  password: 
-  host: 127.0.0.1
+  password:
+  host: <%= ENV['MYSQL_HOST'] || "127.0.0.1" %>
```

Comment:
Probably worth moving host to the different blocks (development, test and production)? dev,test/prod will have different mysql_host for sure.